robots.txt

```diff
@@ -0,0 +1,3 @@
+"User-Agent: *
```

Comment:
I think we dont need the gänsefüßchen (also below)

Reply:
True. Stupid me.